packages/sdk/contractkit/MIGRATION-TO-VIEM.md

```diff
@@ -111,10 +111,9 @@ const registryContract = getContract({
 
 async function CeloTokens(): Promise<[string, string][]> {
   return Promise.all(
-    ['GoldToken', 'StableToken', 'StableTokenEUR', 'StableTokenBRL'].map(async (token) => [
-      token,
-      await registryContract.read.getAddressForString(token),
-    ])
+    ['GoldToken', 'StableToken', 'StableTokenEUR', 'StableTokenBRL', 'StableToken$USDC'].map(
```

Comment:
If I'm not mistaken, this is not going to work for `StableToken$USDC`, because USDC will not be a core contract, so won't be in `Registry.sol`.

Reply:
righto